main.py

```diff
@@ -179,6 +243,7 @@ async def check_videos(self, queue):
                 return
             if response_headers["Content-Type"] == "video/mp4":
                 # exists now, edit message to include link
+                assert isinstance(score["message_id"], int), "Can't attach video to a score with no assigned Discord message"
```

Comment:
Here again if it's in the queue, it will always have a message id. And even if somehow it managed to not have one. I rather it just throws an exception below than bloat up the code with needless statements like this.

Reply:
Ditto.